src/Components/Index/ServiceSelectionComponent.tsx

```diff
@@ -87,41 +83,23 @@ export class ServiceSelectionComponent extends SceneObjectBase<ServiceSelectionC
   }
 
   private _onActivate() {
-    // terribad hack - remove single service filter if it's there
-    const variable = sceneGraph.lookupVariable(VAR_FILTERS, this);
-    if (variable instanceof AdHocFiltersVariable && variable.state.filters.find((f) => f.key === SERVICE_NAME)) {
-      variable.setState({
-        filters: variable.state.filters.filter((f) => f.key !== SERVICE_NAME),
-      });
-    }
```

Comment:
I don't know what was this code doing, but it seems that we don't need it again. 🤷‍♀️